src/Model/Linsen.hs

```diff
@@ -72,6 +69,20 @@ parse day =
           >=> (.: "richText")
           >=> (.: "root")
           >=> (.: "children")
+          >=> (\v' ->
+                 (case v' !? 1 of
+                   Nothing -> fail "failed to index into food"
+                   Just v -> pure v) >>=
+                 withObject "Parse day" (
+                  (.: "children")
+                  >=> (\case
+                          Nothing -> fail "Failed to index into richtext"
+                          Just v -> pure v) . headMay
```

Comment:
Isn't this the same as:
```suggestion
                  >=> \case
                          [] -> fail "Failed to index into richtext"
                          (v:_) -> pure v
```
...which removes one import and doesn't shift over into the realm of Maybes but stays in the list dimension. It's very much up to you how you want to do this though.

Reply:
Yeah, makes sense